### PR TITLE
Stop using wheel packages when installing scipy

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by kmaehashi** [comment]
What is the aim of this PR?
(related to https://github.com/cupy/cupy/pull/909 and https://github.com/chainer/chainer-test/pull/383)

**Comment by okuta** [comment]
Yes. The tester compiles SciPy based on NumPy in the environment.

File: docker.py [hunk 1]
```diff
@@ -521,6 +522,9 @@ def make_dockerfile(conf):
             'RUN pip install -U %s && rm -rf ~/.cache/pip\n' %
             ' '.join(['"%s"' % req for req in requires]))
 
+        if scipy is not None:
```
Comment (unnonouno) [inline docker.py]: Write a comment why it is required.

**Comment by okuta** [comment]
I added comment.

**Comment by okuta** [comment]
@unnonouno Cloud you merge this?

File: docker.py (hunk 1 above)
Comment (unnonouno) [reply docker.py]: Please write detailed description. For example:

SciPy depends on C-API interface of NumPy. When you install different version of NumPy, it breaks compatibility and causes an error. So you need to install SciPy from its source to link NumPy you use.

Comment (okuta) [reply docker.py]: Thanks!

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**